docs/UserGuide.md

```diff
@@ -192,7 +200,8 @@ Notes:
 Finds all persons whose names **and** tags matches ALL keywords provided.
 
 #### Format:
-* `find n/NAME [n/MORE_NAMES]…​ t/TAG [t/MORE_TAGS]…​`
+* `find [n/NAME] [n/MORE_NAMES]…​ [t/TAG] [t/MORE_TAGS]…​`
+  * Note that `find` must have at least one `[n/NAME]` or `[t/TAG]`.
```

Comment:
Abit weird tho cos `[]` means that it is optional, but here you say must have at least one, so its conflicting. I think the original is clearer?